apps/condo/domains/notification/constants/constants.js

```diff
@@ -73,6 +73,7 @@ const RECURRENT_PAYMENT_TOMORROW_PAYMENT_LIMIT_EXCEED_MESSAGE_TYPE = 'RECURRENT_
 const B2C_APP_MESSAGE_PUSH_TYPE = 'B2C_APP_MESSAGE_PUSH'
 const NEWS_ITEM_COMMON_MESSAGE_TYPE = 'NEWS_ITEM_COMMON_MESSAGE_TYPE'
 const NEWS_ITEM_EMERGENCY_MESSAGE_TYPE = 'NEWS_ITEM_EMERGENCY_MESSAGE_TYPE'
+const NEED_TO_PAY_RECEIPT_ON_PAYDAY_MESSAGE_TYPE = 'NEED_TO_PAY_RECEIPT_ON_PAYDAY_MESSAGE'
```

Comment:
Unified name with feature flag will get -1 extra word to remember.

```suggestion
const SEND_BILLING_RECEIPTS_ON_PAYDAY_REMINDER_MESSAGE_TYPE = 'SEND_BILLING_RECEIPTS_ON_PAYDAY_REMINDER_MESSAGE_TYPE'
```